src/Package/Impl/DataInspect/VariableViewModel.cs

```diff
@@ -59,7 +59,7 @@ public sealed class VariableViewModel : RSessionDataObject, IIndexedItem {
             }
 
             var tableCaps = (ViewerCapabilities.Table | ViewerCapabilities.List);
-            CanShowOpenCsv = CanShowDetail && (_detailsViewer.Capabilities & tableCaps) != 0 && result.Length > 0;
+            CanShowOpenCsv = CanShowDetail && result.CanCoerceToDataFrame && (_detailsViewer.Capabilities & tableCaps) != 0 && result.Length > 0;
```

Comment:
While you're here, can you also change the use of `&` to `Enum.HasFlag` for the sake of better clarity, and `tableCaps` to const?